SmartPlaces.Facilities/README.md

```diff
@@ -1,27 +1,70 @@
 # Smart Facilities
 
-This folder contains the code, configuration, and documentation for the Smart Facilities Components. This includes several libraries that are created as Nuget Packages, and some sample code.
+This folder contains the code, configuration, and documentation for the SmartPlaces.Facilities Components. This includes several libraries that are also packaged as Nuget Packages, and some sample code.
```

Comment:
I like "Smart Places Facilities" rather than SmartPlaces.Facilities. This namespace could be found in the code.

Why are we using "also" ? Why not "This includes several libraries that are packaged as Nuget .."